src/views/game/ResultView.vue

```diff
@@ -49,21 +66,61 @@ onMounted(() => {
   const deltaY = windowMiddleY - elementMiddleY
 
   startingTransform.value = `translate( ${deltaX}px, ${deltaY}px)`
-
-  setTimeout(() => {
-    displayTracks.value = true
-  }, 5000 + 2000)
-
-  setTimeout(() => {
-    displayButton.value = true
-  }, 5000 + 2000 + 2000)
 })
 </script>
 
 <template>
-  <div class="grid h-screen grid-cols-[650px_auto] grid-rows-2 place-items-center gap-5">
+  <div v-if="isMobileSize" class="grid place-content-center">
+    <Tabs default-value="leaderboard">
+      <div class="h-10">
+        <Transition name="fade-top">
+          <TabsList v-if="displayTracks" class="w-full">
+            <TabsTrigger value="leaderboard" class="flex-1">
+              Leaderboard
+            </TabsTrigger>
+            <TabsTrigger value="tracks" class="flex-1">
+              Tracks
+            </TabsTrigger>
+          </TabsList>
+        </Transition>
+      </div>
+
+      <TabsContent value="leaderboard" class="h-[60vh] max-h-[60vh] border" force-mount>
+        <GameResults
+          :is-mobile-size
```

Comment:
Pyt: Nie krzyczy Ci tutaj kompilator, że `is` jest undefined? Mi czesto krzyczy jak jak nazwy są kilku członowe

Reply:
Nope